webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js

```diff
@@ -206,11 +206,17 @@ export class SakaiRubricGrading extends RubricsElement {
 
   fineTuneRating(e) {
 
-    var value = e.target.value;
+    const value = e.target.value;
+
+    const parsed = parseFloat(value.replace(/,/g, "."));
```

Comment:
is this going to work if browser is in ES locale ?

Reply:
Yep, parseFloat only works on decimals. That's why I convert commas to decimals here.